app/services/dqt/v1/teacher.rb

```diff
@@ -14,8 +14,12 @@ def self.find(trn:, birthdate:, nino: nil)
         }
         query[:nino] = nino if nino
 
+        Rails.logger.info("DQT API request started")
```

Comment:
I guess this is after the day as those will trickle in but not instantly?

Reply:
I have a snippet to check the integration on the day which will verify the keys/secrets work, but this is just an extra check to ensure we can monitor requests are actually happening/successful